requirements-dev.txt

```diff
@@ -6,10 +6,9 @@ pylatexenc>=1.4
 stestr>=2.0.0
 ddt>=1.2.0,!=1.4.0,!=1.4.3
 reno>=3.4.0
-Sphinx>=1.8.3,!=3.1.0,!=5.2.0,!=5.2.0.post0
+Sphinx>=5.0
```

Comment:
This removes the penalty against `5.2.0` and `5.2.0.post0`. I don't think that was intentional was it?

Reply:
I did this based on simplifying it down to what Terra it updated to. I think at the time we had issues with those versions. I do not know if that still is the case. Since its not what we pull into CI any longer I figured why maintain an older list of exceptions here that I do not know if we hit problems with any more or not. If someone has that old version and finds a problem in building locally they can simply update it, It does not break our CI here anymore (since we install 5.3.0), which is more why these restrictions were added.